packages/rocketchat-mapview/server/settings.coffee

```diff
@@ -0,0 +1,3 @@
+Meteor.startup ->
+	RocketChat.settings.add 'MapView_Enabled', false, {type: 'boolean', group: 'Message', section: 'Google Maps', public: true, i18nLabel: 'MapView Enabled', i18nDescription: 'MapView_Enabled_Description'}
```

Comment:
I've updated it with keys, and added descriptions properly. I'm fairly sure I don't need to touch the other i18n files, as i only added my keys to the en.i18n.json file.